### PR TITLE
Fix Android crash by initializing flag

### PR DESCRIPTION
**Pull request by Damnum**
Fixes https://github.com/zmxv/react-native-sound/issues/367

### CODE REVIEW

**Comment by trepidity** [comment]
@Damnum Thanks for this PR.
Your PR includes changes to syntax/formating that isn't required for your PR. Would you mind undoing that? (see my inline comments)

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java [hunk 1]
```diff
@@ -202,7 +202,7 @@ public void play(final Integer key, final Callback callback) {
     // Request audio focus in Android system
     if (!this.mixWithOthers) {
       AudioManager audioManager = (AudioManager) context.getSystemService(Context.AUDIO_SERVICE);
-      
+
```
Comment (trepidity) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java]: Please undo this.

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java [hunk 2]
```diff
@@ -248,7 +248,7 @@ public void pause(final Integer key, final Callback callback) {
     if (player != null && player.isPlaying()) {
       player.pause();
     }
-    
+
```
Comment (trepidity) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java]: Please undo this.

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java [hunk 3]
```diff
@@ -261,7 +261,7 @@ public void stop(final Integer key, final Callback callback) {
       player.pause();
       player.seekTo(0);
     }
-    
+
```
Comment (trepidity) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java]: Please undo this.

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java [hunk 4]
```diff
@@ -389,7 +389,7 @@ public void setCategory(final String category, final Boolean mixWithOthers) {
   public void onAudioFocusChange(int focusChange) {
     if (!this.mixWithOthers) {
       MediaPlayer player = this.playerPool.get(this.focusedPlayerKey);
-      
+
```
Comment (trepidity) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java]: Please undo this.

**Comment by Damnum** [comment]
@trepidity ok done

**Comment by ofirdagan** [comment]
following

**Comment by Progoogler** [comment]
This fixes my problem with #372. Thank you for your contribution!

**Comment by ofirdagan** [comment]
Thanks for that! Any chance to release a new version?

**Comment by trepidity** [comment]
published 0.10.6 today.

**Pull request merged**